randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -23,13 +23,17 @@ Extra - asset_id: 3289414871
   * Normal Door to Transport to Magmoor Caverns West/Door to Shoreline Entrance
   * Extra - dock_index: 0
   > Door to Phendrana Shorelines
-      Charge Beam or Missile ≥ 3
+      Any of the following:
+          Charge Beam
```

Comment:
Shouldn't this also need any of the beams?

Reply:
Good catch